index.html

```diff
@@ -31,6 +31,160 @@ <h1>Roland<br> Ossisa Yuma</h1>
+                            <ul class="flex ul-format">
+                                <li class="courses">HTML/CSS</li>
+                                <li class="courses">Ruby on Rails</li>
+                                <li class="courses">JavaScript</li>
+                            </ul>
+                            <div class="clearfix"></div>
+                            <button class="project-button flex flex-align">See this project <img src="img/arrow.svg" alt=""></button>
+                        </div>
+                        <div class="clearfix-big"></div>
+                    </div>
+                </div>
+                <div class="card">
+                    <img class="project-pic-one" src="img/project3.svg" alt="">
+                    <div class="project-name-one">
+                        <div class="set">
+                            <h3>Project name goes here</h3>
+                            <ul class="flex ul-format">
+                                <li class="courses">HTML/CSS</li>
+                                <li class="courses">Ruby on Rails</li>
+                                <li class="courses">JavaScript</li>
+                            </ul>
+                            <div class="clearfix"></div>
+                            <button class="project-button flex flex-align">See this project <img src="img/arrow.svg" alt=""></button>
+                        </div>
+                        <div class="clearfix-big"></div>
+                    </div>
+                </div>
+                <div class="card">
+                    <img class="project-pic-one" src="img/project4.svg" alt="">
+                    <div class="project-name-one">
+                        <div class="set">
+                            <h3>Project name goes here</h3>
+                            <ul class="flex ul-format">
+                                <li class="courses">HTML/CSS</li>
+                                <li class="courses">Ruby on Rails</li>
+                                <li class="courses">JavaScript</li>
+                            </ul>
+                            <div class="clearfix"></div>
+                            <button class="project-button flex flex-align">See this project <img src="img/arrow.svg" alt=""></button>
+                        </div>
+                        <div class="clearfix-big"></div>
+                    </div>
+                </div>
+                <div class="card">
+                    <img class="project-pic-one" src="img/project5.svg" alt="">
+                    <div class="project-name-one">
+                        <div class="set">
+                            <h3>Project name goes here</h3>
+                            <ul class="flex ul-format">
+                                <li class="courses">HTML/CSS</li>
+                                <li class="courses">Ruby on Rails</li>
+                                <li class="courses">JavaScript</li>
```

Comment:
- [ ] [OPTIONAL] Consider removing the padding-left on the list of technology as it may help improve your design
![Uploading Screen Shot 2022-03-18 at 16.03.48.png…]()